canonical-websites/insights.ubuntu.com
# Updated snappy logo for the snappy topic page

## Done

* Updated snappy logo for the snappy topic page
* Added a new variable `topic.strip_css` to the topics.json and topics.html template to allow us to override the exiting (and default) `p-strip`

# QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Open this page [snappy topic page](http://0.0.0.0:8023/topics/snappy/)
- see that the snappy icon is https://assets.ubuntu.com/v1/a861450c-snapcraft-logo.svg

## Issue / Card

Fixes #74

## Screenshot

![image](https://user-images.githubusercontent.com/441217/34640893-3a38590a-f2f3-11e7-86b6-9dbeb585c442.png)
